#This message allows to configure the pixel-meter mapping for the robot's mirror
float32 max_distance
float32 step
uint8 filter_lines
uint16[] pixel_distances
uint16[] lines_length
position[] mask_contour

